latest_cov_build.py

```diff
@@ -59,19 +70,26 @@
 if not os.path.exists('tools'):
     os.makedirs('tools')
 
-# Download
-name = os.path.join('tools', version)
-base_url += version
-urlretrieve(base_url, name)
-
-# Extract
-if version.endswith('zip'):
-    with zipfile.ZipFile(name, 'r') as zip_ref:
-        zip_ref.extractall('tools')
-
-# MacOS and Linux OS have 'tar.gz' extensions
-elif base_url.endswith('tar.gz'):
-    with tarfile.open(name, 'r:gz') as tar:
+# Download geckodriver
+gecko_archive = os.path.join('tools', version)
+gecko_url += version
+urlretrieve(gecko_url, gecko_archive)
+
+# Download grcov
+grcov_archive = os.path.join('tools', 'grcov.tar.bz2')
+urlretrieve(grcov_url, grcov_archive)
+
+# Extract and delete archives
+for filename in [gecko_archive, grcov_archive, 'tools/target.tar.bz2']:
+    if filename.endswith('zip'):
+        with zipfile.ZipFile(filename, 'r') as zip_ref:
+            zip_ref.extractall('tools')
+        os.remove(filename)
+        continue
```

Comment:
This is a bit ugly, the `os.remove(filename)` is already after the if-else branches, so we don't need to call it here and then `continue`.

Reply:
Thanks for comments.
Here, I wanted to finish the loop iteration if `.zip` file was found not to go to part with `tarfile`; therefore, I've added continue.  
I'll change it.